.gitignore

```diff
@@ -14,6 +14,7 @@
 *.nuget.props
 *.nuget.targets
 NuGet.exe
+nuget.exe
```

Comment:
nuget.exe shouldn't be used at all. The toolset uses msbuild or dotnet cli to restore. How does nuget show up?